frontend/src/features/properties/filter/PropertyFilter.tsx

```diff
@@ -153,9 +155,10 @@ export const PropertyFilter: React.FC<IPropertyFilterProps> = ({
                   setFieldValue('name', getIn(newValues[0], 'value') ?? newValues[0]);
                 }}
                 ref={ref}
-                onBlur={(e: any) =>
-                  getIn(values, 'name') === '' && setFieldValue('name', e.target.value)
-                }
+                onBlur={(e: any) => {
```

Comment:
Found another bug where name was not updating with free form text on subsequent requests, now solved with the following changes